src/quickgui/attributes/qgsquickattributeformmodelbase.cpp

```diff
@@ -260,7 +262,7 @@ void QgsQuickAttributeFormModelBase::flatten( QgsAttributeEditorContainer *conta
         item->setData( QStringLiteral( "field" ), QgsQuickAttributeFormModel::ElementType );
         item->setData( fieldIndex, QgsQuickAttributeFormModel::FieldIndex );
         item->setData( container->isGroupBox() ? container->name() : QString(), QgsQuickAttributeFormModel::Group );
-        item->setData( true, QgsQuickAttributeFormModel::CurrentlyVisible );
+        item->setData( fieldName != QStringLiteral( "uuid" ) && fieldName != QStringLiteral( "fid" ), QgsQuickAttributeFormModel::CurrentlyVisible );
```

Comment:
we are here in QGIS library, not sure why there should be hardcoded "uuid" or "fid" or anything like that.